src/deroff.rs

```diff
@@ -1,352 +1,277 @@
-//         'lh': "<=",
-//         'rh': "=>",
-//         'tf': "therefore",
-//         '~~': "~~",
-//         '~=': "~=",
-//         '!=': "!=",
-//         '**': '*',
-//         '+-': '\261',
-//         '<=': "<=",
-//         '==': "==",
-//         '=~': "=~",
-//         '>=': ">=",
-//         'AN': "\\/",
-//         'OR': "/\\",
-//         'no': '\254',
-//         'te': "there exists",
-//         'fa': "for all",
-//         'Ah': "aleph",
-//         'Im': "imaginary",
-//         'Re': "real",
-//         'if': "infinity",
-//         'md': "\267",
-//         'mo': "member of",
-//         'mu': '\327',
-//         'nm': "not member of",
-//         'pl': '+',
-//         'eq': '=',
-//         'pt': "oc",
-//         'pp': "perpendicular",
-//         'sb': "(=",
-//         'sp': "=)",
-//         'ib': "(-",
-//         'ip': "-)",
-//         'ap': '~',
-//         'is': 'I',
-//         'sr': "root",
-//         'pd': 'd',
-//         'c*': "(x)",
-//         'c+': "(+)",
-//         'ca': "cap",
-//         'cu': 'U',
-//         'di': '\367',
-//         'gr': 'V',
-//         'es': "{}",
-//         'CR': "_|",
-//         'st': "such that",
-//         '/_': "/_",
-//         'lz': "<>",
-//         'an': '-',
-//
-//         # Output Greek
-//         '*A': "Alpha",
-//         '*B': "Beta",
-//         '*C': "Xi",
-//         '*D': "Delta",
-//         '*E': "Epsilon",
-//         '*F': "Phi",
-//         '*G': "Gamma",
-//         '*H': "Theta",
-//         '*I': "Iota",
-//         '*K': "Kappa",
-//         '*L': "Lambda",
-//         '*M': "Mu",
-//         '*N': "Nu",
-//         '*O': "Omicron",
-//         '*P': "Pi",
-//         '*Q': "Psi",
-//         '*R': "Rho",
-//         '*S': "Sigma",
-//         '*T': "Tau",
-//         '*U': "Upsilon",
-//         '*W': "Omega",
-//         '*X': "Chi",
-//         '*Y': "Eta",
-//         '*Z': "Zeta",
-//         '*a': "alpha",
-//         '*b': "beta",
-//         '*c': "xi",
-//         '*d': "delta",
-//         '*e': "epsilon",
-//         '*f': "phi",
-//         '+f': "phi",
-//         '*g': "gamma",
-//         '*h': "theta",
-//         '+h': "theta",
-//         '*i': "iota",
-//         '*k': "kappa",
-//         '*l': "lambda",
-//         '*m': "\265",
-//         '*n': "nu",
-//         '*o': "omicron",
-//         '*p': "pi",
-//         '+p': "omega",
-//         '*q': "psi",
-//         '*r': "rho",
-//         '*s': "sigma",
-//         '*t': "tau",
-//         '*u': "upsilon",
-//         '*w': "omega",
-//         '*x': "chi",
-//         '*y': "eta",
-//         '*z': "zeta",
-//         'ts': "sigma",
-//     }
+struct Deroffer {}
+
+impl Deroffer {
+    // for the moment, return small strings, until we figure out what
+    // it should really be doing
+    fn g_specs_specletter(key: &str) -> Option<&'static str> {
+        Some(match key {
+            // Output composed latin1 letters
+            "-D" => &"\u{00d0}",
```

Comment:
Ah...kk...played with Python2 vs Python3 on the command line. Makes sense now for these to be character codes rather than raw byte codes. Didn't realize that the Python 2 vs 3 encoding difference would apply to string escape sequences (e.g., `'\320'` as in this case).

:+1:

Reply:
That's a good point, I'll include it in this PR :-)